application/src/main/java/uk/nhs/hee/tis/revalidation/mapper/RecommendationViewMapper.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright 2022 Crown Copyright (Health Education England)
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this software and
+ * associated documentation files (the "Software"), to deal in the Software without restriction,
+ * including without limitation the rights to use, copy, modify, merge, publish, distribute,
+ * sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all copies or
+ * substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT
+ * NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+ * DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package uk.nhs.hee.tis.revalidation.mapper;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import uk.nhs.hee.tis.revalidation.dto.MasterDoctorViewDto;
+import uk.nhs.hee.tis.revalidation.entity.RecommendationView;
+
+@Mapper(componentModel = "spring")
+public interface RecommendationViewMapper {
+  @Mapping(target = "designatedBody", source = "masterDoctorViewDto.designatedBody")
+  @Mapping(target = "gmcStatus", source = "masterDoctorViewDto.gmcStatus")
+  @Mapping(target = "tisStatus", source = "masterDoctorViewDto.tisStatus")
```

Comment:
Is there any reason we want to do the special mapping for `designatedBody`, `gmcStatus`, `tisStatus`?